wsmaster/che-core-api-ssh/src/test/java/org/eclipse/che/api/ssh/server/jpa/SshJpaTckRepository.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.ssh.server.jpa;
+
+import com.google.inject.persist.Transactional;
+
+import org.eclipse.che.api.ssh.server.model.impl.SshPairImpl;
+import org.eclipse.che.api.user.server.model.impl.UserImpl;
+import org.eclipse.che.commons.test.tck.repository.TckRepository;
+import org.eclipse.che.commons.test.tck.repository.TckRepositoryException;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+import javax.persistence.EntityManager;
+import java.util.Collection;
+
+import static java.util.Collections.emptyList;
+
+/**
+ * @author Mihail Kuznyetsov
+ */
+@Transactional
+public class SshJpaTckRepository implements TckRepository<SshPairImpl> {
+    @Inject
+    private Provider<EntityManager> managerProvider;
+
+    @Override
+    public void createAll(Collection<? extends SshPairImpl> sshPairs) throws TckRepositoryException {
+        EntityManager manager = this.managerProvider.get();
+        for (SshPairImpl sshPair : sshPairs) {
+            if (manager.find(UserImpl.class, sshPair.getOwner()) == null) {
```

Comment:
I'd create all ssh pair related object in before method of Test like https://github.com/sleshchenko/che/blob/accounts/wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/spi/tck/WorkspaceDaoTest.java#L97
Then you should not check user existence and tck repository will looks very simple https://github.com/sleshchenko/che/blob/accounts/wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/jpa/WorkspaceTckModule.java#L46-L47